frontend/public/components/app.jsx

```diff
@@ -203,18 +204,21 @@ class App_ extends React.PureComponent {
     );
 
     return (
-      <DetectPerspective>
-        <DetectNamespace>
-          {contextProviderExtensions.reduce(
-            (children, e) => (
-              <EnhancedProvider key={e.uid} {...e.properties}>
-                {children}
-              </EnhancedProvider>
-            ),
-            content,
-          )}
-        </DetectNamespace>
-      </DetectPerspective>
+      <>
```

Comment:
Why do we need a react fragment here?

Reply:
Since we are rendering the `DetectLanguage` here with the `DetectPerspective`, it needs a parent element.
I didn't include children in `DetectLanguage` since it is only meant to run once on first load.